pages/_app.js

```diff
@@ -12,11 +12,14 @@ function MyApp({ Component, pageProps }) {
     'savedWine',
     initialWineState
   );
-
+  const [boughtWine, setBoughtWine] = useLocalStorage(
+    'boughtWine',
+    initialWineState
+  );
   return (
     <SnackbarProvider
       anchorOrigin={{
-        vertical: 'bottom',
+        vertical: 'top',
```

Comment:
Usually, you'd want notifications to stack at the bottom left for example. Is there any particular reason you've chosen to put the notifications at the top? If it overlaps with your navbar, perhaps you can investigate in the Notistack doc if there's an attribute you could use to change the positioning a little. : )